oxidation/crates/parsec_api_types/src/addr.rs

```diff
@@ -211,24 +212,22 @@ impl BaseBackendAddr {
         url
     }
 
-    pub fn to_http_redirection_url(&self) -> Url {
+    /// Create a url for http request.
+    pub fn to_http_url(&self) -> Url {
```

Comment:
I think we said we could remove entirely this and only use `BaseBackendAddr::to_http_url_with_path(None)` instead

Reply:
I was asking myself, if I wouldn't change its (for `to_http_url_with_path`) signature to

```rust
pub fn to_http_url_with_path(&self, path: &str) -> Url {}
```

Reply:
> my comment above was for the case when we have both to_http_url and to_http_url_with_path

I think it's better to only have a single `to_http_url_with_path(path: Option<...>)`

Reply:
Done, I've merged both